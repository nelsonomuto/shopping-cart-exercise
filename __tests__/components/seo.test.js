import { shallow } from "enzyme"
import React from "react"
import SEO from "../../src/components/seo"

describe("<SEO />", () => {
  it("matches snapshot", () => {
    const component = shallow(<SEO title="TEST" />)
    expect(component).toMatchSnapshot()
  })
})
