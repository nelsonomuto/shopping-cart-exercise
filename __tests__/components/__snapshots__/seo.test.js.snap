// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<SEO /> matches snapshot (with keywords) 1`] = `
<HelmetWrapper
  defer={true}
  encodeSpecialCharacters={true}
  htmlAttributes={
    Object {
      "lang": "en",
    }
  }
  meta={
    Array [
      Object {
        "content": "test",
        "name": "keywords",
      },
    ]
  }
  title="TEST"
/>
`;

exports[`<SEO /> matches snapshot 1`] = `
<HelmetWrapper
  defer={true}
  encodeSpecialCharacters={true}
  htmlAttributes={
    Object {
      "lang": "en",
    }
  }
  meta={Array []}
  title="TEST"
/>
`;
